runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerPlugin.java

```diff
@@ -55,31 +53,138 @@
 import net.runelite.http.api.worlds.WorldType;
 import net.runelite.http.api.xp.XpClient;
 
```

Comment:
No newline here, and you should configure your editor to not move stuff around as it sees fit in the future.